### ISSUE TITLE
Can not resize a newly inserted image after deleting any other image from the editor.

### ISSUE THREAD

**Issue by magda-chrzescian** [ckeditor/ckeditor5#10156]
## 📝 Provide detailed reproduction steps (if any)

1. Enter the image-resize demo on `master`.
2. Remove an image.
3. Insert a new image (no insert button in the demo, use pasting).
4. It is not possible to resize an image - it just stays the same size.

![2021-07-14 17 48 14](https://user-images.githubusercontent.com/14334820/125652827-634b810d-e77a-4bf8-9492-0e8759b37f55.gif)

```
ckeditorerror.js:69 Uncaught TypeError: Cannot read property 'contains' of undefined
    at Resizer.containsHandle (resizer.js:293)
    at WidgetResize._getResizerByHandle (widgetresize.js:191)
    at WidgetResize._mouseDownListener (widgetresize.js:209)
    at Object.fire (emittermixin.js:208)
    at fireListenerFor (bubblingemittermixin.js:168)
    at Document.fire (bubblingemittermixin.js:98)
    at MouseObserver.fire (domeventobserver.js:100)
    at MouseObserver.onDomEvent (mouseobserver.js:31)
    at ProxyEmitter.listenTo.useCapture (domeventobserver.js:83)
    at ProxyEmitter.fire (emittermixin.js:208)
```
It got broken after merging the content minimap MVP1 https://github.com/ckeditor/ckeditor5/pull/10090, there were some changes in the `packages/ckeditor5-widget/src/widgetresize/resizer.js` file.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
The fix is simple

```diff
diff --git a/packages/ckeditor5-widget/src/widgetresize/resizer.js b/packages/ckeditor5-widget/src/widgetresize/resizer.js
index d9ff9137fa..83e9993fb3 100644
--- a/packages/ckeditor5-widget/src/widgetresize/resizer.js
+++ b/packages/ckeditor5-widget/src/widgetresize/resizer.js
@@ -281,7 +281,7 @@ export default class Resizer {
          }
 
          containsHandle( domElement ) {
-                  return this._domResizerWrapper.contains( domElement );
+                  return this._domResizerWrapper && this._domResizerWrapper.contains( domElement );
          }
```

Story time: In my changes for the minimap I needed to avoid caching `this._domResizerWrapper` because it can be rendered multiple times in editing view clones (minimap clones the editing view and it renders like a view). After my changes, it could be that `this._domResizerWrapper` (returned on demand) does not exist if the resizer was removed. At the same time `WidgetResize` plugin **never removes entries** from its internal `this._resizers` map so it may happen that it iterates over all resizers (it does that on mouse down) and `Resizer.containsHandle` is called for a resizer that has been removed (removed means no `_domResizerWrapper`).

The **alternative fix** could clean `WidgetResize#_resizers` up to keep only these resizers that actually exist in the editing. It's also worth checking because this looks very dirty to me (the map is growing and growing as users insert and remove images).
